plugins/server/org.mongodb/mongodb/2.2/manifest.ktor.yaml

```diff
@@ -0,0 +1,13 @@
+name: MongoDB
+description: Adds MongoDB database to your application
+vcsLink: https://github.com/mongodb/mongo
+license: Apache 2.0
+category: Databases
+prerequisites:
+  - routing
+  - kotlinx-serialization
+documentation: documentation.md
+installation:
+  outside_app: Databases.kt
```

Comment:
Sorry, I just realized that this should actually be `default: install.kt` (or any filename) and the templating system will move it to the `Databases.kt` automatically and call it from the application module.